### PR TITLE
Portfolio Contact Form

### PR DESCRIPTION
**Pull request by Suigeneriz**
We created the following:
- A contact form

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -527,3 +527,102 @@ body {
...
+  height: 813px;
+  left: 0;
+  top: 4288px;
+  background: #f9f9fa;
+}
+
+.contact-form {
+  padding: 0 20px;
+}
+
```
Comment (Ugboaja-Uchechi) [inline styles.css:543]: - [ ] Your webpage is overflowing. Kindly fix this by giving if an `overflow-x: hidden;` 🙏 

20220705-225524_capture

File: styles.css [hunk 2]
```diff
@@ -527,3 +527,102 @@ body {
...
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
+}
+
+.form-area {
+  text-align: center;
+  margin: 0 auto;
+}
+
```
Comment (Ugboaja-Uchechi) [inline styles.css:568]: - [ ] Kindly make sure your design looks as close to the Figma design as possible. 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: styles.css [hunk 3]
```diff
@@ -527,3 +527,102 @@ body {
   bottom: 74.49%;
   opacity: 0.5;
 }
+
+.contact-section {
+  position: absolute;
+  width: 100%;
```
Comment (Ugboaja-Uchechi) [inline styles.css:533]: - [ ] Please make sure you layout your design using `flexbox` as stated in the project requirement 🙏

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Suigeneriz  and @kendoriddy , 

## Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights

- Linters are passing ✔️ 
- Repository looks professional 👍 
- Great job with your design 👏 
- Form is being submitted successfully 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Suigeneriz) [reply styles.css]: Fixed

File: styles.css (hunk 2 above)
Comment (Suigeneriz) [reply styles.css]: Fixed

File: styles.css (hunk 3 above)
Comment (Suigeneriz) [reply styles.css]: Fixed

**Pull request merged**